app/controllers/admin/address_types_controller.rb

```diff
@@ -1,90 +1,70 @@
+      respond_with @address_types
     end
-  end
 
-  # GET /address_types/1
-  # GET /address_types/1.json
-  def show
-    @address_type = AddressType.find(params[:id])
-
-    respond_to do |format|
-      format.html # show.html.erb
-      format.json { 
-        render json: @address_type 
-      }
+    def show
+      respond_with @address_type
     end
-  end
 
-  # GET /address_types/new
-  # GET /address_types/new.json
-  def new
-    @address_type = AddressType.new
+    def new
+      @address_type = AddressType.new
+      respond_with @address_type
+    end
 
-    respond_to do |format|
-      format.html # new.html.erb
-      format.json { render json: @address_type }
+    def edit
+      respond_with @address_type
     end
-  end
 
-  # GET /address_types/1/edit
-  def edit
-    @address_type = AddressType.find(params[:id])
-  end
+    def create
+      @address_type = AddressType.new(params[:address_type])
 
-  # POST /address_types
-  # POST /address_types.json
-  def create
-    @address_type = AddressType.new(params[:address_type])
+      respond_to do |format|
+        if @address_type.save
+          message = 'Address type was successfully created.'
+          format.html { redirect_to edit_admin_address_type_path(@address_type), notice: message }
+          format.json { render json: @address_type, status: :created, location: address_type_link }
+        else
+          render_error_response(format, template: :new, model: @address_type)
+        end
+      end
+    end
 
-    respond_to do |format|
-      if @address_type.save
-        purge_all_pages
-        format.html { redirect_to edit_admin_address_type_path(@address_type), notice: 'Address type was successfully created.' }
-        format.json { render json: @address_type, status: :created, location: admin_address_types_url(@address_type) }
-      else
-        format.html { render action: "new" }
-        format.json { render json: @address_type.errors, status: :unprocessable_entity }
+    def update
+      respond_to do |format|
+        if @address_type.update_attributes(params[:address_type])
+          message = 'Address type was successfully updated.'
+          format.html { redirect_to edit_admin_address_type_path(@address_type), notice: message }
+          format.json { head :no_content }
+        else
+          render_error_response(format, template: :edit, model: @address_type)
+        end
       end
     end
-  end
 
-  # PUT /address_types/1
-  # PUT /address_types/1.json
-  def update
-    @address_type = AddressType.find(params[:id])
+    def destroy
+      @address_type.destroy
 
-    respond_to do |format|
-      if @address_type.update_attributes(params[:address_type])
-        purge_all_pages
-        format.html { redirect_to edit_admin_address_type_path(@address_type), notice: 'Address type was successfully updated.' }
+      respond_to do |format|
+        format.html { redirect_to admin_address_types_url }
```

Comment:
@leoapost I just checked and that diff is wrong :) I did change that line but the value is still a path. You can see the context if you display the whole file. There were lines reduced so that probably the reason why git diff looks off.